### PR TITLE
Add the ability to define the Crowdin branch name manually

### PR DESCRIPTION
**Pull request by CosmaTrix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,14 @@ In your `package.json`, add these scripts:
...
 ```
 
+## Branch name
+
+By default the `@mollie/crowdin-in` uses the current git branch name for the syncing to the Crowdin branch. It is possible to override that by defining a `CROWDIN_BRANCH_NAME` environment variable in the `.env` file
+
+```shell
+CROWDIN_BRANCH_NAME={your-branch-name}
+```
+
```
Comment (rppld) [inline README.md:38]: Shouldn’t this rather be a flag for the respective command, rather than an env var?

Comment (mattijsbliek) [reply README.md:38]: Same here, would expect this to be `--branch=master`. The tool has Crowdin in the name, so I think we can be concise 😊

Comment (CosmaTrix) [reply README.md]: Thanks. What you both say makes perfect sense. I've changed it to be instead a `--branch-name` (or `-b`) option that you can pass to the `upload`, `download` and `delete-branch` commands.
```shell
mollie-crowdin download --branch-name my-custom-branch-name
```

File: README.md [hunk 2]
```diff
@@ -28,6 +28,14 @@ In your `package.json`, add these scripts:
 "crowdin:delete-branch": "mollie-crowdin delete-branch"
 ```
 
+## Branch name
+
+By default the `@mollie/crowdin-cli` uses the current git branch name for the syncing to the Crowdin branch. It is possible to override that by passing a `--branch-name` option (or `-b` ) to the `upload`, `download` and `delete-branch` commands. For example, the following command will collect and upload the translations to a branch named `custom-branch-name`.
```
Comment (rppld) [inline README.md:33]:
```suggestion
By default, `@mollie/crowdin-cli` will create a branch in Crowdin with the same name as the currently checked out branch in Git. It is possible to override this behaviour by passing a `--branch-name` option (or `-b` ) to the `upload`, `download` and `delete-branch` commands. For example, the following command will collect and upload messages to a branch named `custom-branch-name` in Crowdin.
```

File: README.md [hunk 3]
```diff
@@ -28,6 +28,14 @@ In your `package.json`, add these scripts:
 "crowdin:delete-branch": "mollie-crowdin delete-branch"
 ```
 
+## Branch name
```
Comment (rppld) [inline README.md:31]: In order to not confuse it with the Git branch name

```suggestion
## Crowdin branch name
```

File: src/cli.ts [hunk 4]
```diff
@@ -1,12 +1,17 @@
...
 import deleteBranch from "./delete-branch";
+import config from "./config";
 
 const main = async (argv: string[]) => {
-  const program = new commander.Command();
+  const program = new Command();
   const version = require("../package.json").version;
+  const branchOption = new Option(
+    "-b, --branch-name [string]",
+    "the Crowdin branch name where to sync the translations"
```
Comment (rppld) [inline src/cli.ts:13]:
```suggestion
    "the name of the branch to be created in Crowdin"
```

**Review by rppld — APPROVED** [review]
Nice! Thanks for tidying up the options/config stuff. Couple copy remarks, but already approving 👍

File: src/cli.ts (hunk 4 above)
Comment (rppld) [reply src/cli.ts:13]: Think you can ignore this one, as it won’t apply when used in the `download` command.

**Review by rppld — APPROVED** [review]

**Comment by el-angel** [comment]
@CosmaTrix why do we want this?

**Pull request merged**